src/components/organisms/NavBarSchedule/NavBarSchedule.tsx

```diff
@@ -205,13 +229,43 @@ export const NavBarSchedule: React.FC<NavBarScheduleProps> = ({
     "NavBarSchedule--show": isVisible,
   });
 
+  const isNotSovereignVenue = venue.id !== sovereignVenue?.id;
+  const breadcrumbSovereignVenueClasses = classNames(
+    "NavBarScheduleBreadcrumb__btn",
+    {
+      "NavBarScheduleBreadcrumb__btn--disabled": filterRelatedEvents,
+    }
+  );
+
+  const breadcrumbVenueClasses = classNames("NavBarScheduleBreadcrumb__btn", {
+    "NavBarScheduleBreadcrumb__btn--disabled": !filterRelatedEvents,
+  });
+
   return (
     <div className="NavBarSchedule__wrapper">
       <div className={containerClasses}>
         {/* Disabled as per designs. Up for deletion if confirmied not necessary */}
         {/* {<ScheduleVenueDescription />} */}
 
         <ul className="NavBarSchedule__weekdays">{weekdays}</ul>
+        <div className="NavBarSchedule__breadcrumb">
+          <label>Events on: </label>
+          <button
+            onClick={() => setFilterRelatedEvents(false)}
+            className={breadcrumbSovereignVenueClasses}
+          >
+            {sovereignVenue?.name}
+          </button>
+          /
+          {isNotSovereignVenue && (
+            <button
+              onClick={() => setFilterRelatedEvents(true)}
+              className={breadcrumbVenueClasses}
+            >
+              {venue?.name}
+            </button>
```

Comment:
It looks like this button could be extracted like `<BreadcrumbButton`. 
And the logic of true/false as setFilterRelatedEvents's argument depends on `isNotSovereignVenue` value

Reply:
created a general Breadcrumbs component here: https://github.com/sparkletown/sparkle/pull/2352/commits/90366891c725a5ed617297b35a2917d624ca566a